### ISSUE TITLE
Out-of-date reference to drush.ini

### ISSUE THREAD

**Issue by dabley** [drush-ops/drush#4411]
**Describe the bug**
When Drush is called with invalid php parameters, preflight checks in src/Preflight/PreflightVerify.php produce the error message:

> 'Please check your configuration settings in !phpini or in your drush.ini file; see examples/example.drush.ini for details.'

However, the file examples/example.drush.ini no longer exists in the repo. As reported in #3294, Drush 9 no longer makes use of drush.ini files. Therefore, the preflight error message should be modified to avoid giving this impression.

**To Reproduce**
```
$ mkdir /tmp/drushrc
$ echo 'disable_functions=system' > /tmp/drushrc/drush.ini
$ PHP_INI_SCAN_DIR=:/tmp/drushrc ../vendor/drush/drush/drush st
```

**Expected behavior**
I expect the error message to give up-to-date and helpful information. Since "Drush 9 is now out of the business of setting up your PHP environment." (according to https://github.com/drush-ops/drush/issues/3294#issuecomment-356345090), it would be helpful if the preflight error message could give a pointer towards information on how the PHP environment can actually be set up.

**Actual behavior**
Error message is:
>  [preflight] The following restricted PHP modes have non-empty values: {disable_functions}. This configuration is incompatible with drush.  {Please check your configuration settings in /etc/php/7.3/cli/php.ini or in your drush.ini file; see examples/example.drush.ini for details.}

**Workaround**
Ignore the out-of-date reference to drush.ini and example.drush.ini. See #3294 for alternative ways to configure the php environment, by explicitly setting PHPRC or PHP_INI_SCAN_DIR environment variables.

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 10.2.2
| Drupal version? | 8.8.5
| PHP version     | 7.3
| OS?             | Linux

**Comment by weitzman** [comment]
Actually, I wonder if we need any of this code anymore. We rely on Symfony Process to execute processes now and it uses popen() so I dont think we care anymore about those disabled_functions. @greg-1-anderson - what do you think about removing https://github.com/drush-ops/drush/blob/fb48fd4942d31bc27b60b5183761613a191fa4ed/src/Preflight/PreflightVerify.php#L57-L77 in drush 10?

**Comment by greg-1-anderson** [comment]
I think that `safe_mode` and `open_basedir` probably still cause problems for Drush. I'd be surprised if `exec` or `system` **didn't** appear in at least one of our vendor libraries. Since we are relying more heavily on `popen`, if anything I would think we would add that check to this list rather than remove the others.

I can't say for sure, though. A long time ago we used to get a lot of support requests from folks who had Drush running on systems that blocked the use of `exec`. We don't see those any more. Is that because this code is working, and people fix their .ini values without bugging us? I'd guess, though, that if providers are still using disable_functions, they'd probably add popen if they add exec. Maybe not though.

I'd rather convert from a hard error to a warning than go straight to removing the code. However, there's no b/c break to removing it, so if you want to test the waters and take it out, and see if support requests increase, I'm game. We could always put it back mid-major version as a bugfix if it turns out to be a problem.

**Comment by weitzman** [comment]
Closed in #4502

Looks like Process already checks for proc_open https://github.com/symfony/process/blob/2cf3f1373f84bdd33a3dfb1fb99eaa29371859af/Process.php#L142-L144
